src/modules/data/blocks/rsvp/sagas.js

```diff
@@ -66,16 +68,16 @@ export function* setRSVPTempDetails( action ) {
 }
 
 export function* initializeRSVP() {
-	const start = yield select( blocks.datetime.selectors.getStart );
-	const end = yield select( blocks.datetime.selectors.getEnd );
+	const publishDate = wpSelect( 'core/editor' ).getEditedPostAttribute( 'date' );
+	const eventEnd = yield select( blocks.datetime.selectors.getEnd );
```

Comment:
shouldn't this be `eventStart` and `selectors.getStart`?

Reply:
That's what I originally thought too, but according to this ticket it's supposed to be the publish date of the event initially

Reply:
https://central.tri.be/issues/117844

Reply:
had a conversation with gustavo earlier, he confirmed it's the end date.

Reply:
Ah shoot, I misread. Fixing

Reply:
Fixed here: https://github.com/moderntribe/event-tickets/pull/912